std_msgs/Header header                                        # Includes measurement timestamp and coordinate frame.
OdomMsg odom_msg
FusionPoints[] radar_points                            # Includes many radar fused points